lib/tasks/publishing_api.rake

```diff
@@ -68,21 +69,55 @@ namespace :publishing_api do
 
     [
       {
-        title: 'GDS Design Principles',
-        description: '',
-        content_id: '8b7b20d4-4144-4f52-922a-2fec92e1a346',
-        base_path: '/design-principles',
+        title: "GDS Design Principles",
+        content_id: "52028eb3-c0ef-412d-a986-2fc0a342569e",
+        base_path: "/design-principles/mainstream",
+      },
+      {
+        title: "GDS Design Principles",
+        content_id: "a82e1adb-637d-4cfe-986b-b65dd7fe4f48",
+        base_path: "/design-principles/insidegovernment",
+      },
+      {
+        title: "GDS Design Principles",
+        content_id: "93554c24-c13c-49ed-b5de-759a4cb0a7b8",
+        base_path: "/design-principles/detailedguides",
+      },
+      {
+        title: "GDS Design Principles",
+        content_id: "aa5431bb-2f2e-4e6a-a819-d25c5ba6b5d0",
+        base_path: "/design-principles/accessiblepdfs",
+      },
+      {
+        title: "GDS Design Principles",
+        content_id: "74a68f1b-d86d-42e4-b79d-05ff5b932265",
+        base_path: "/design-principles/performanceframework",
+      },
+      {
+        title: "GDS Design Principles",
+        content_id: "185e6998-106e-494f-87fd-46d095ec32dc",
+        base_path: "/design-principles/whatsnew",
+      },
+      {
+        title: "GDS Design Principles",
+        content_id: "ce017d8c-7d39-4a8c-aed9-af236edd656a",
+        base_path: "/design-principles/seo",
+      },
+      {
+        title: "GDS Design Principles",
+        content_id: "7672ae7d-6efb-4816-bd74-c49a6cb43f04",
+        base_path: "/design-principles/style-guide.atom",
       },
       {
         title: 'Transformation programme',
-        description: '',
         content_id: '00573998-7254-407c-ba28-776dd222cda3',
         base_path: '/transformation',
+        type: "prefix",
       }
     ].each(&helper.method(:publish_special_route))
 
     [
-      # Redirections because the slugs changed.
+      %w(8b7b20d4-4144-4f52-922a-2fec92e1a346 /design-principles exact /guidance/government-design-principles),
       %w(b9b74acd-58bd-4e6b-8318-e6926a25c84d /designprinciples prefix /design-principles),
```

Comment:
Should this go directly to /guidance/government-design-principles as well, to avoid a double hop?

Reply:
I figured no because then wouldn't we need to set up all the same exact routes but with `/designprinciples` at the start instead?

Reply:
Fair enough.